File: quests/qeynos/Nostahla_Westfend.pl
sub EVENT_WAYPOINT_ARRIVE {
  if($wp == 3) {
	$npc->SetAppearance(1);
    	}
  if($wp == 4) {
    $npc->SetAppearance(0);
	}
  if($wp == 16) {
    quest::say("Gahlith, I'm going down to the docks to see if Ghul has anything new from overseas today. I'll be back in a little while.");
	}
}



sub EVENT_SAY { 
if($text=~/Hail/i){
quest::say("Greetings..  I am Nostahla Westfend, understudy of Gahlith, and Second Master Wizard of the Order of Three."); }
}
#END of FILE Zone:qeynos  ID:1129 -- Nostahla_Westfend 

$npc->SetAppearance(1);

File: quests/qeynos/Nostahla_Westfend.lua
function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("Greetings..  I am Nostahla Westfend, understudy of Gahlith, and Second Master Wizard of the Order of Three.");
	end
end

function event_waypoint_arrive(e)
	if(e.wp == 3) then
		eq.set_anim(1121,1);
	elseif(e.wp == 4) then
		eq.set_anim(1121,0);
	elseif(e.wp == 16) then
		e.self:Say("Gahlith, I'm going down to the docks to see if Ghul has anything new from overseas today. I'll be back in a little while.");
	end
end
